ui/src/tool-panel/location-track/location-track-location-infobox.tsx

```diff
@@ -125,9 +125,16 @@ export const LocationTrackLocationInfobox: React.FC<LocationTrackLocationInfobox
     );
 
     const getSplittingDisabledReasonsTranslated = () => {
-        const reasons = [];
-        if (!publishTypeIsDraft)
-            reasons.push(t('tool-panel.disabled.activity-disabled-in-official-mode'));
+        const reasons: string[] = [];
+
+        if (!publishTypeIsDraft) {
+            return t('tool-panel.disabled.activity-disabled-in-official-mode');
+        }
+
+        if (extraInfo?.partOfUnfinishedSplit) {
+            return t('tool-panel.location-track.splitting-blocks-geometry-changes');
```

Comment:
Tämä virheilmoitus on muotoa "Raiteen geometriaa ei voi muokata koska sen jakaminen on kesken." Tuo sanamuoto ei mielestäni ole kauhean oikein, sillä monesti näitä näytetään jakamisen lopputuoteraiteille. Alkuperäiselle jaettavalle raiteelle tuo olisi ihan hyvä. Onko parempia ehdotuksia?